lms/templates/certificates/_about-edx.html

```diff
@@ -1,10 +1,11 @@
+<%page expression_filter="h"/>
 <%! from django.utils.translation import ugettext as _ %>
 <section class="about-item about-edx">
     <h2 class="about-edx-title hd-4">${company_about_title}</h2>
 
     <div class="about-edx-copy copy copy-meta">
         <p>${company_about_description}</p>
 
-        <a class="action" href="${company_about_url}">${company_about_urltext}</a>
+        <a class="action" href="${company_about_url | n, decode.utf8}">${company_about_urltext}</a>
```

Comment:
According to http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/preventing_xss.html#url-context-and-escaping, we still need to HTML/Javascript-escape URLs that are added to the page. Therefore, we can remove the n filter here.